compiler/rustc_middle/src/mir/query.rs

```diff
@@ -440,4 +448,28 @@ impl<'tcx> TyCtxt<'tcx> {
             self.promoted_mir(def.did)
         }
     }
+
+    #[inline]
+    pub fn optimized_mir_of_opt_const_arg(
```

Comment:
Some of these are suffixed with `of_opt_const_arg` while some have `opt_const_arg`. Does this denote different semantics? If not, can we use the same suffix for everything?

Reply:
removes `of` for all of them